packages/jazz-tools/src/tools/subscribe/SubscriptionScope.ts

```diff
@@ -525,6 +589,32 @@ export class SubscriptionScope<D extends CoValue> {
       this.subscribers.forEach((listener) => listener(value));
     }
 
+    if (!this.firstLoadRecorded) {
+      trackPerformanceMark("subscriptionLoadEnd", this.id);
```

Comment:
This might not catch some subscriptions load time correctly.

We arrive here only if there are listeners, so e.g. when the values are already loaded we would end up recording the first update.

I think it should be placed above, [maybe here](https://github.com/garden-co/jazz/blob/8c59660e0383270159194195f62f624be6f43c24/packages/jazz-tools/src/tools/subscribe/SubscriptionScope.ts#L579) but not too sure.

We probably need a different approach or a refactoring of SubscriptionScope to have some more explicit lifecycles.